recipe/meta.yaml

```diff
@@ -1,17 +1,17 @@
 {% set name = "appimage-updater-bridge" %}
-{% set version = "1.1.9" %}
+{% set version = "2.0.0" %}
 
 package:
   name: {{ name }}
   version: {{ version }}
 
 source:
   url: https://github.com/antony-jr/AppImageUpdaterBridge/archive/v{{ version }}.tar.gz
-  sha256: 9dcb8dace1d6e385794d9e859f1a9e228c9bdfac649d2f2fcb1641d1a3177f8a
+  sha256: 529ec0acfc834b544b09f70f1873ba3411e8f151c6248f28d59c07f86f72e2a1
```

Comment:
@looooo This should be ```7ca47d659bb504f188a9683775249febb970624e157684d6750f5d77b562f8af``` because I've deleted the original v2x release and re-published it. Sorry about this.